src/applications/pensions/containers/UnauthenticatedWarningAlert.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { toggleLoginModal as toggleLoginModalAction } from 'platform/site-wide/user-nav/actions';
+
+const UnauthenticatedWarningAlert = ({ toggleLoginModal, isLoggedIn }) => {
+  const showLoginModal = e => {
+    e.preventDefault();
+    toggleLoginModal(true);
+  };
+
+  if (isLoggedIn) {
+    return null;
+  }
+
+  return (
+    <va-alert status="warning" uswds slim>
+      <React.Fragment key=".1">
+        <p className="vads-u-margin-y--0">
+          This application is 7 steps long and it contains several substeps per
+          step. We advise you{' '}
+          {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
+          <a href="#" onClick={showLoginModal}>
```

Comment:
Need to use a button here instead of a link per the VA Design system

Reply:
Is there a specific reason we need to use a button instead of a link? I'll mention this to the design team as I see both Alert links and buttons used in the VADS.